modules/frontend/manifests/packages.pp

```diff
@@ -8,14 +8,14 @@
   exec { "nodepackages":
     path    => "/bin:/usr/bin",
     unless  => "npm list -g --depth=0 | grep grunt-cli",
-    command => "npm install -g coffee-script bower grunt-cli gulp component yo",
```

Comment:
Yeah @andrewmallis i would just throw in whatever your most basic global deps are here